### DISCUSSION TITLE
Emails not working after changing nameserver

### DISCUSSION THREAD

**Discussion by arpit3210** [vercel/community#4638] [Help] [answered]
### Summary

![image](https://github.com/vercel/community/assets/75988023/b18bbd86-d4d3-49a9-909f-b63c39dc0b6d)

### Steps to Reproduce

I have changed the nameserver of the domain, and after this my email is not working, what should do?

**Comment by amyegan** [comment]
If you share the domain, I can help you investigate the problem.

**Reply by arpit3210** [reply]
Email-         [ valerio.jacobi@myweb3gift.com  ]   
Domain-             www.myweb3gift.com

**Reply by swarnava** [reply]
Did you set the MX, SPF, DKIM or other necessary record from Vercel dashboard?

**Reply by arpit3210** [reply]
I'm confused, about which mx DNS preset records need to be selected, I added Outlook first, didn't work. 

https://github.com/orgs/vercel/discussions/4638#discussion-5782887

**Reply by swarnava** [reply]
You don't need to set any Preset, just mX record and recommended value (provided by your email provider) should work

**Reply by arpit3210** [reply]
I got myweb3gift.com domain from www.strato.de

strato dashboard showing  MX-Record -  inactive


There is no option to check, which MX record needs to be selected.

![image](https://github.com/vercel/community/assets/75988023/5a795267-f0cb-4ef5-8298-aca1dd9cd53b)

**Reply by swarnava** [reply]
Unfortunately, this is beyond scope of our knowledge. I would recommend reaching out to Strato support. :)

**Reply by arpit3210** [reply]
@swarnava, can we schedule a support call with your team member to resolve the issue?

**Reply by swarnava** [reply, marked as answer]
Hi Arpit, due to nature of our distributed support team, we are unable to provide phone support. As the DNS is managed by Strato, it will be better if you connect with their support team. Thank you!

**Reply by arpit3210** [reply]
I understand your time is valuable, but I must mention that the problem at hand is still unresolved. It's been more than a month since I first opened this issue, and despite my efforts, I haven't been able to figure out a solution. I even reached out to the Strato team for assistance and I was advised to contact your support team so I can share some confidential details related to the issue. However, I cannot disclose these details publicly. So please, could you provide me with further support to resolve this matter?

**Comment by diceboi** [comment]
same problem here. I even added the mx record to vercel from my original nameserver, waiting to get some emails.

**Reply by arpit3210** [reply]
Solved ?

**Reply by diceboi** [reply]
Nope unfortunatelly. Still searching for any kind of answer.

**Reply by swarnava** [reply]
Did you set the MX, SPF, DKIM or other necessary record from Vercel dashboard?

**Reply by diceboi** [reply]
I added the recommended A record and CNAME record to verify the domain when I lauched the page. Then my email stopped working. After that I tried to add the mx record to vercel domain dashboard. I need to add DKIM and SFP to vercel dashboard aswell?

**Reply by arpit3210** [reply]
@diceboi  Issue solved ?

**Reply by swarnava** [reply]
Hi @diceboi, could you create a separate discussion thread so we can triage separately? Thank you!

**Reply by diceboi** [reply]
@arpit3210 @swarnava My domain provider resolved this issue in my case, maybe it helps. 
They modified the base MX record that was: example.com, and the modified version changed to mail.example.com in my case. Then they added an A record (with the original A record IP adress, the one that was before the 76.76 vercel ip) to that mail.example.com. This resolved the issue.
